src/bin/cargo/commands/owner.rs

```diff
@@ -6,46 +6,137 @@ use cargo_credential::Secret;
 pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
-        .arg(Arg::new("crate").value_name("CRATE").action(ArgAction::Set))
+        .arg_quiet()
+        .arg(Arg::new("crate").hide(true))
+        .arg_required_else_help(true)
+        .args_conflicts_with_subcommands(true)
+        .override_usage(
+            "\
+       cargo owner add    <OWNER_NAME> [CRATE_NAME] [OPTIONS]
+       cargo owner remove <OWNER_NAME> [CRATE_NAME] [OPTIONS]
+       cargo owner list   [CRATE_NAME] [OPTIONS]",
+        )
+        // The following three parameters are planned to be replaced in the form of subcommands.
+        // refer to issue: https://github.com/rust-lang/cargo/issues/4352
```

Comment:
This comment speaks of future tense but isn't it doing that in this PR?

Reply:
My intention is to remind other contributors that the --add/--list/--remove parameters are temporarily reserved and will be gradually discarded in the future.

If you think it's redundant, I'll delete it.

Reply:
We can't remove the arguments.  They might be deprecated; we might hide them, but we can't remove them.